llama_index/storage/kvstore/mongodb_kvstore.py

```diff
@@ -130,6 +130,13 @@ async def aput(
 
         """
         raise NotImplementedError
+    
+    def put_all(self, kv_pairs: List[Tuple[str, dict]], collection: str = DEFAULT_COLLECTION, batch_size: int = DEFAULT_BATCH_SIZE) -> None:
```

Comment:
can you add async func for this as well

Reply:
Unfortunately i don't have the time to properly test the async func for put_all if i implemented it. Can we merge this individually?